app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,80 @@
+<section class='section'>
```

Comment:
Please add a toggle button to set the recipe public on the `recipes show` page as it's given in [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_food.png?token=GHSAT0AAAAAAB3BA6QVUC66BJTL6LJFWWQSY3NUFOA) wireframe.